public_html/wp-content/plugins/pattern-creator/pattern-creator.php

```diff
@@ -13,7 +13,9 @@
 namespace WordPressdotorg\Pattern_Creator;
 use const WordPressdotorg\Pattern_Directory\Pattern_Post_Type\POST_TYPE;
 
+const AUTOSAVE_INTERVAL = 30;
 const QUERY_VAR = 'edit-pattern';
+const PATTERN_ID_VAR = 'patternId';
```

Comment:
Typically keys like this use kebab-case in PHP.

```suggestion
const PATTERN_ID_VAR = 'pattern-id';
```